components/database/mariadb-106/test/results-all.master

```diff
@@ -435,7 +439,7 @@ connect.odbc_xls [ skipped ] Need ODBC data source ConnectEngineXLS
 connect.part_file [ pass ]
 connect.part_table [ pass ]
 connect.pivot [ pass ]
-connect.rest [ pass ]
+connect.rest [ skipped ] Need Curl or Casablanca
```

Comment:
@fritzkink do you really have no curl installed?  Or, do you have it, but test failed to find it?  I'm curious how this got skipped...

Thank you.

Reply:
Thats a good question. I've curl installed. I tried to find out how this test is working. Currently my guess is that this file 

COMPONENT_TEST_ARGS += --skip-test-list=$(COMPONENT_DIR)/files/disabled-tests

contains the answer despite the test in question is not explicitly mentioned. To be honest I was not aware of this file before just found it because of your question.

Reply:
BTW this is the start of the file build/test/amd64/test-64-results

`Logging: /usr/src/oi-userland/components/database/mariadb-106/mariadb-10.6.17/mysql-test/mariadb-test-run.pl  --parallel=8 --mem --force --retry=0 --force-restart --max-test-fail=0 --skip-test-list=/usr/src/oi-userland/components/database/mariadb-106/files/disabled-tests`

Reply:
> Currently my guess is that this file
> 
> COMPONENT_TEST_ARGS += --skip-test-list=$(COMPONENT_DIR)/files/disabled-tests
> 
> contains the answer despite the test in question is not explicitly mentioned.

I do not think so.  I tested 10.6.16 with that file and the test passed.

> To be honest I was not aware of this file before just found it because of your question.

It is new so it is not surprising you missed it.

Reply:
> BTW this is the start of the file build/test/amd64/test-64-results
> 
> `Logging: /usr/src/oi-userland/components/database/mariadb-106/mariadb-10.6.17/mysql-test/mariadb-test-run.pl --parallel=8 --mem --force --retry=0 --force-restart --max-test-fail=0 --skip-test-list=/usr/src/oi-userland/components/database/mariadb-106/files/disabled-tests`

I do not see anything unexpected there...